docs/lib/analytics/fragments/flutter/getting-started/10_preReq.md

```diff
@@ -3,3 +3,9 @@
     * An iOS configuration targeting at least iOS 11.0
     * An Android configuration targeting at least Android API level 21 (Android 5.0) or above
     * For a full example of please follow the [project setup walkthrough](~/lib/project-setup/create-application.md)
+
+<amplify-callout warning>
+
+If you are using Flutter (2.2.0), you need to disable sound null safety by following the instructions [here](https://dart.dev/null-safety/unsound-null-safety#testing-or-running-mixed-version-programs). Null safety support for Amplify Flutter is being worked on actively.
```

Comment:
non-blocking nit: Does it make sense to change this to "(2.2.0 or greater)" to clarify that it's not this specific version of flutter, but all the versions after (presumably)? Maybe this is obvious to most readers but a small clarification.

Reply:
nm, I see this merged